### PR TITLE
Fixes deprecation warning in MiniGo

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

**Comment by BradLarson** [comment]
You may have seen this due to a slight lag in the availability of new toolchains, but the use of scalars for `epsilon` and `momentum` was just changed yesterday in PR #220 and a matching PR in swift-apis: https://github.com/tensorflow/swift-apis/pull/525

This is by design, and should pass compilation once the new swift-apis changes are in the nightly toolchains.

File: MiniGo/Models/GoModel.swift [hunk 1]
```diff
@@ -54,7 +54,7 @@ struct ConvBN: Layer {
         // TODO(jekbradbury): thread through bias and affine boolean arguments
         // (behavior is correct for inference but this should be changed for training)
         self.conv = Conv2D(filterShape: filterShape, strides: strides, padding: padding)
-        self.norm = BatchNorm(featureCount: filterShape.3, momentum: 0.95, epsilon: 1e-5)
+        self.norm = BatchNorm(featureCount: filterShape.3, momentum: Tensor<Float>(0.95), epsilon: Tensor<Float>(1e-5))
```
Comment (dan-zheng) [inline MiniGo/Models/GoModel.swift]: Could you please revert this change? As https://github.com/tensorflow/swift-models/pull/221#issuecomment-551912299 explains: `momentum` and `epsilon` now have type `Scalar`.

Ubuntu nightly toolchains should already have this change.
A new macOS toolchain will be released with this change soon.

---

The other deprecation warning fix is valid!

Comment (valeriyvan) [reply MiniGo/Models/GoModel.swift]: done

**Comment by dan-zheng** [comment]
CI shows [more warnings](https://source.cloud.google.com/results/invocations/83f4f236-1cf2-45bf-a71f-f034b2d88462/targets/s4tf%2Fmodels%2Fubuntu%2Fpresubmit/log), could you please fix those as well?
```
/swift-models/Transformer/PythonCheckpointReader.swift:39:12: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
    return Raw.restoreV2(prefix: StringTensor(path),
           ^
/swift-models/Transformer/PythonCheckpointReader.swift:39:12: note: use '_Raw' instead
    return Raw.restoreV2(prefix: StringTensor(path),
           ^~~
           _Raw
/swift-models/Transformer/PythonCheckpointReader.swift:76:22: error: cannot convert value of type 'Tensor<Double>' to expected argument type 'Scalar'
            epsilon: Tensor(1e-5))
                     ^
                                  as! Scalar
[18/69] Merging module Gym-Blackjack
[19/69] Compiling MiniGo MCTSNode.swift
[20/69] Compiling MiniGo MCTSRandomPredictor.swift
[23/69] Compiling MiniGo GoModel.swift
/swift-models/MiniGo/Models/GoModel.swift:57:21: error: cannot invoke initializer for type 'BatchNorm<_>' with an argument list of type '(featureCount: Int, momentum: Tensor<Float>, epsilon: Tensor<Float>)'
        self.norm = BatchNorm(
                    ^
/swift-models/MiniGo/Models/GoModel.swift:57:21: note: expected an argument list of type '(featureCount: Int, axis: Int, momentum: Scalar, epsilon: Scalar)'
        self.norm = BatchNorm(
                    ^
[24/69] Compiling MiniGo MCTSPredictor.swift
[25/69] Compiling MiniGo MCTSModelBasePredictor.swift
/swift-models/MiniGo/Strategies/MCTS/MCTSModelBasePredictor.swift:122:30: warning: 'transposed(withPermutations:)' is deprecated: renamed to 'transposed(permutation:)'
        return featureTensor.transposed(withPermutations: 1, 2, 0)
                             ^
/swift-models/MiniGo/Strategies/MCTS/MCTSModelBasePredictor.swift:122:30: note: use 'transposed(permutation:)' instead
        return featureTensor.transposed(withPermutations: 1, 2, 0)
                             ^~~~~~~~~~ ~~~~~~~~~~~~~~~~
                             transposed permutation
[27/69] Compiling MiniGo PythonCheckpointReader.swift
/swift-models/MiniGo/Models/PythonCheckpointReader.swift:31:16: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        return Raw.restoreV2(prefix: StringTensor(path),
               ^
/swift-models/MiniGo/Models/PythonCheckpointReader.swift:31:16: note: use '_Raw' instead
        return Raw.restoreV2(prefix: StringTensor(path),
               ^~~
               _Raw
/swift-models/Transformer/main.swift:56:14: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
    tokens = Raw.multinomial(logits: lastLogit.squeezingShape(at: 1), numSamples: Tensor<Int32>(1))
             ^
/swift-models/Transformer/main.swift:56:14: note: use '_Raw' instead
    tokens = Raw.multinomial(logits: lastLogit.squeezingShape(at: 1), numSamples: Tensor<Int32>(1))
             ^~~
             _Raw
/swift-models/Datasets/CIFAR10/CIFAR10.swift:116:44: warning: 'transposed(withPermutations:)' is deprecated: renamed to 'transposed(permutation:)'
    let imageTensor = Tensor<Float>(images.transposed(withPermutations: [0, 2, 3, 1]))
                                           ^
/swift-models/Datasets/CIFAR10/CIFAR10.swift:116:44: note: use 'transposed(permutation:)' instead
    let imageTensor = Tensor<Float>(images.transposed(withPermutations: [0, 2, 3, 1]))
                                           ^~~~~~~~~~ ~~~~~~~~~~~~~~~~
                                           transposed permutation
/swift-models/Datasets/CIFAR10/CIFAR10.swift:128:16: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        label: Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.label }),
               ^
/swift-models/Datasets/CIFAR10/CIFAR10.swift:128:16: note: use '_Raw' instead
        label: Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.label }),
               ^~~
               _Raw
/swift-models/Datasets/CIFAR10/CIFAR10.swift:129:15: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        data: Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.data })
              ^
/swift-models/Datasets/CIFAR10/CIFAR10.swift:129:15: note: use '_Raw' instead
        data: Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.data })
              ^~~
              _Raw
[31/70] Merging module Gym-CartPole
[32/70] Wrapping AST for Gym-CartPole for debugging
/swift-models/Transformer/Model.swift:95:16: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
    let mask = Raw.matrixBandPart(
               ^
/swift-models/Transformer/Model.swift:95:16: note: use '_Raw' instead
    let mask = Raw.matrixBandPart(
               ^~~
               _Raw
/swift-models/Transformer/Model.swift:141:37: warning: 'transposed(withPermutations:)' is deprecated: renamed to 'transposed(permutation:)'
    let movedToFront = splitLastDim.transposed(withPermutations: 0, 2, 1, 3)
                                    ^
/swift-models/Transformer/Model.swift:141:37: note: use 'transposed(permutation:)' instead
    let movedToFront = splitLastDim.transposed(withPermutations: 0, 2, 1, 3)
                                    ^~~~~~~~~~ ~~~~~~~~~~~~~~~~
                                    transposed permutation
/swift-models/Transformer/Model.swift:152:37: warning: 'transposed(withPermutations:)' is deprecated: renamed to 'transposed(permutation:)'
    let movedToBack = splitFirstDim.transposed(withPermutations: 0, 2, 1, 3)
                                    ^
/swift-models/Transformer/Model.swift:152:37: note: use 'transposed(permutation:)' instead
    let movedToBack = splitFirstDim.transposed(withPermutations: 0, 2, 1, 3)
                                    ^~~~~~~~~~ ~~~~~~~~~~~~~~~~
                                    transposed permutation
/swift-models/Transformer/Model.swift:176:16: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        return Raw.concatV2([seed.query, seed.key, seed.value], axis: Tensor<Int32>(2))
               ^
/swift-models/Transformer/Model.swift:176:16: note: use '_Raw' instead
        return Raw.concatV2([seed.query, seed.key, seed.value], axis: Tensor<Int32>(2))
               ^~~
               _Raw
/swift-models/Transformer/Model.swift:230:77: error: cannot convert value of type 'Tensor<Float>' to expected argument type 'Float'
        selfAttentionNorm = LayerNorm(featureCount: size, axis: 2, epsilon: Tensor<Float>(1e-5))
                                                                            ^
/swift-models/Transformer/Model.swift:233:75: error: cannot convert value of type 'Tensor<Float>' to expected argument type 'Float'
        feedForwardNorm = LayerNorm(featureCount: size, axis: 2, epsilon: Tensor<Float>(1e-5))
                                                                          ^
[35/71] Linking Gym-CartPole
[37/71] Merging module Catch
/swift-models/Support/Image.swift:40:26: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        let loadedFile = Raw.readFile(filename: StringTensor(url.absoluteString))
                         ^
/swift-models/Support/Image.swift:40:26: note: use '_Raw' instead
        let loadedFile = Raw.readFile(filename: StringTensor(url.absoluteString))
                         ^~~
                         _Raw
/swift-models/Support/Image.swift:41:26: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        let loadedJpeg = Raw.decodeJpeg(contents: loadedFile, channels: 3, dctMethod: "")
                         ^
/swift-models/Support/Image.swift:41:26: note: use '_Raw' instead
        let loadedJpeg = Raw.decodeJpeg(contents: loadedFile, channels: 3, dctMethod: "")
                         ^~~
                         _Raw
/swift-models/Support/Image.swift:44:23: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
                data: Raw.reverse(loadedJpeg, dims: Tensor<Bool>([false, false, false, true])))
                      ^
/swift-models/Support/Image.swift:44:23: note: use '_Raw' instead
                data: Raw.reverse(loadedJpeg, dims: Tensor<Bool>([false, false, false, true])))
                      ^~~
                      _Raw
/swift-models/Support/Image.swift:62:27: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        let encodedJpeg = Raw.encodeJpeg(
                          ^
/swift-models/Support/Image.swift:62:27: note: use '_Raw' instead
        let encodedJpeg = Raw.encodeJpeg(
                          ^~~
                          _Raw
/swift-models/Support/Image.swift:64:9: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
        Raw.writeFile(filename: StringTensor(url.absoluteString), contents: encodedJpeg)
        ^
/swift-models/Support/Image.swift:64:9: note: use '_Raw' instead
        Raw.writeFile(filename: StringTensor(url.absoluteString), contents: encodedJpeg)
        ^~~
        _Raw
/swift-models/Support/Image.swift:71:25: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
                tensor: Raw.resizeBilinear(
                        ^
/swift-models/Support/Image.swift:71:25: note: use '_Raw' instead
                tensor: Raw.resizeBilinear(
                        ^~~
                        _Raw
/swift-models/Support/Image.swift:76:25: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
                tensor: Raw.resizeBilinear(
                        ^
/swift-models/Support/Image.swift:76:25: note: use '_Raw' instead
                tensor: Raw.resizeBilinear(
                        ^~~
                        _Raw
[39/72] Wrapping AST for Gym-FrozenLake for debugging
[40/72] Wrapping AST for Catch for debugging
[41/72] Wrapping AST for Gym-Blackjack for debugging
[42/72] Merging module ModelSupport
[43/72] Wrapping AST for ModelSupport for debugging
[44/72] Linking Gym-FrozenLake
[45/72] Linking Gym-Blackjack
[46/72] Linking Catch
/swift-models/Datasets/MNIST/MNIST.swift:107:22: warning: 'transposed(withPermutations:)' is deprecated: renamed to 'transposed(permutation:)'
                    .transposed(withPermutations: [0, 2, 3, 1]) / 255,  // NHWC
                     ^
/swift-models/Datasets/MNIST/MNIST.swift:107:22: note: use 'transposed(permutation:)' instead
                    .transposed(withPermutations: [0, 2, 3, 1]) / 255,  // NHWC
                     ^~~~~~~~~~ ~~~~~~~~~~~~~~~~
                     transposed permutation
[48/73] Merging module Datasets
/swift-models/Transformer/Operators.swift:40:12: warning: 'Raw' is deprecated: 'Raw' has been renamed to '_Raw' to indicate that it is not a guaranteed/stable API.
    return Raw.batchMatMul(left, right, adjX: adjointLeft, adjY: adjointRight)
           ^
/swift-models/Transformer/Operators.swift:40:12: note: use '_Raw' instead
    return Raw.batchMatMul(left, right, adjX: adjointLeft, adjY: adjointRight)
           ^~~
           _Raw


[ID: 8456322] Build finished after 244 secs, exit value: 1


Warning: Permanently added 'localhost' (ECDSA) to the list of known hosts.
```

**Comment by BradLarson** [comment]
To make things easier to pull this in, I've gone ahead and addressed the other deprecation warnings in PR #223. Once we bring these two in, that should take care of those warnings.
